src/app/shared/search/filters/search-filters.component.ts

```diff
@@ -108,11 +106,4 @@ export class SearchFiltersComponent implements OnInit {
     return this.selected.jurisdiction === null  ||
            this.selected.jurisdiction === undefined;
   }
-
-  private selectCaseType(caseTypes: CaseType[]) {
-    if (caseTypes.length === 1) {
-      this.selected.caseType = caseTypes[0];
-      this.onCaseTypeIdChange();
```

Comment:
do we call this one now?

Reply:
nope, we select the first case type in the list as default behaviour